### ISSUE TITLE
Query mode TEXT returns no data for Last Value

### ISSUE THREAD

**Issue by Positronico** [alexanderzobnin/grafana-zabbix#1062]
**Describe the bug**
A query mode **Text** doesn't return the Last Value. When used in a Stat panel formatted as table, it shows No Data. When used in a Table panel formatted as a table, the Last Value is empty. Formatted as a Time Series, the time column for the collected data is correct, but the value column is empty.

**Expected behavior**
It was working as a Stat "legacy" mode, showing the last collected value and also as in table mode, on Grafana 7.1.5. Problem started after 7.2.0 update.

**Screenshots**
![image](https://user-images.githubusercontent.com/23188958/94330563-5c6d9580-ff7a-11ea-9b63-8f27abba00d7.png)
![image](https://user-images.githubusercontent.com/23188958/94330570-6db6a200-ff7a-11ea-9799-45b654f6156a.png)

**Network data**
![image](https://user-images.githubusercontent.com/23188958/94330628-18c75b80-ff7b-11ea-8ae2-990cc8839494.png)

```
{"result":[{"clock":"1601074858","itemid":"34820","ns":"752173290","value":"x.x.x.x"},
{"clock":"1601074923","itemid":"34820","ns":"874368151","value":"x.x.x.x"},
{"clock":"1601074988","itemid":"34820","ns":"524686158","value":"x.x.x.x"},
(...)
{"clock":"1601096419","itemid":"34820","ns":"485776228","value":"x.x.x.x"}]}
```

**Software versions**

```
| Grafana | Zabbix | Grafana-Zabbix Plugin |
| ------- | ------ | --------------------- |
| 7.2.0   | 4.4.10 | 4.0.1                 |
```

**Comment by AhmadNourXV** [comment]
I'm having this issue as well. I reverted to 7.1.5 and the issue is fixed.

**Comment by roman-tiukh** [comment]
I have the same

**Comment by bastun-del** [comment]
I have the same problem with guarana 7.2.0 and Zabbix-server  5.0

**Comment by borjasueiro** [comment]
I have de same issue.

- **Grafana 7.2.0**
- **Plugin 4.0.1**
- Zabbix 5.0.4

**Comment by adrianomazi** [comment]
I have the same issue.

-     Grafana 7.2.0
-     Plugin 4.0.1
-     Zabbix 5.0.4

**Comment by alexanderzobnin** [comment]
There's an issue in grafana repo: https://github.com/grafana/grafana/issues/28042

**Comment by isaqueprofeta** [comment]
I have the same issue.

- Grafana 7.2.1
- Plugin 4.0.1
- Zabbix 4.4.10

**Comment by Positronico** [comment]
I think we have to reopen this one.

Ref.: https://github.com/grafana/grafana/issues/27971

**Comment by alexanderzobnin** [comment]
I'm working on switching to the [Data Frame](https://grafana.com/docs/grafana/latest/developers/plugins/data-frames/) format, so I think it should fix this issue.

**Comment by alexanderzobnin** [comment]
Ah, sorry. That's probably a replace variables issue. Could you try to set `.*` value to the text filter field? If it works in this case, then issue is fixed in 6c20b07c69c7e5ef611aaa6b69f8eb678f77bb2e and will be released soon.

**Comment by cawhite6** [comment]
I can confirm the catch-all (.*) in filter populated the string value.....

![Screen Shot 2020-11-09 at 9 13 42 AM](https://user-images.githubusercontent.com/15822207/98552358-5bfc3600-226c-11eb-8f62-e2bc3b769fed.png)

**Comment by alexanderzobnin** [comment]
Ok, that means bug is on the plugin side and fix will be included in the next release.

**Comment by alexanderzobnin** [comment]
Fixed in 6c20b07c69c7e5ef611aaa6b69f8eb678f77bb2e
